test/passing/override.ml.ref

```diff
@@ -0,0 +1,9 @@
+let _ = {<x = (x : t)>}
+
+let _ = {<x = ((x [@a]) : (t[@b])) [@c]>}
+
+let _ = {<x>}
+
+let _ = {<x = (x [@a])>}
```

Comment:
These parentheses are optional but I think it's fine to keep them for readability. We need to keep track of this kind of decisions we take somewhere.